### DISCUSSION TITLE
Existing BS Data Editor - things we DO like

### DISCUSSION THREAD

**Discussion by amis92** [BSData/phalanx#12] [Ideas]
Please add any and all the *good* features of the existing BS data editor.

**Comment by FarseerVeraenthis** [comment]
The fact it comes bundled with the roster editor

**Comment by cartag** [comment]
Two main resizable columns to the UI, item selection or folder tree similar structure to the left and data on the right. 
Ability to type in fields for Categories or Target to locate items quicker. If there were an easier method devised, I'd be content with that. 
Ability to easily collapse groups

**Comment by cartag** [comment]
Like the right click menu options "Entry Link" but suggest expanding to two entries, SSE Link and SSEG Link.
![image](https://user-images.githubusercontent.com/4388344/143323334-78fa3bf0-2e40-460b-bf7b-5eb43d017256.png)

Like Usages option right click to find where things are used

**Comment by cartag** [comment]
Do like Revision Not Incremented reminder
![image](https://user-images.githubusercontent.com/4388344/143324812-195442d9-de55-4503-8849-aaf344270a28.png)

**Comment by cartag** [comment]
I do like that you can mark things as hidden, even if they don't actually go hidden in BS as it stands right now. Categories for one.
